src/content/protocols/sky.md

```diff
@@ -0,0 +1,381 @@
+
+The `Spotter` feeds the price used for liquidation and auctions. It reads the price out of the `OSM` (Oracle Security Module) which delays price updates by one hour and can be frozen in case of emergency. Incorrect oracle prices could potentially liquidate the entire Maker ecosystem, the oracle provider Chronicle is therefore extremely trusted.
+
+The `Pot` and `Jug` contracts represent the rates module. Respectively the `Pot`contract allows user to save at a specific _DAI Savings rate_ and the `Jug` contract handles stability fees. We note that the savings rate is not limited and may become either highly negative or positive under future governance proposals.
+
+As for the governance and tokens, the Mom contracts (`FlapperMom`, `FLopperMom`, `OSMMom`) have the ability to stop their child. The permissions to do so are granted by `DSChief`, the governance contract, so that governance proposals can stop those contracts without delay.
+
+![Overview of the stablecoin module](./diagrams/sky-stablecoin.png)
+
+## Sky ↔ Maker Conversion Module
+
+The contracts related to the conversion from legacy Maker tokens to Sky are displayed in the diagram below. Following the _Sky End Game Plan_, the `DAI` and `MKR` have been replaced by `USDS` and `SKY` respectively. Conversion contracts allow for the swap from legacy to new tokens and back at a fixed rate. In addition to that, a `LitePSM` (Peg-stability-module) allows users to swap `USDC` for `USDS` with no impact on the peg thanks to pre-attributed reserves.
+
+A `LitePSM` (Peg-stability-module) allows users to swap other stablecoins for `USDS` with no impact on the peg thanks to pre-attributed reserves.
+
+The governance has admin privileges over all contracts and could, for example, arbitrarily mint tokens. `DSChief`, the (non-delayed) governance contract, is the authority over the `LitePSMMom` which allows governance proposals to stop the `LitePSM` without any delay.
+
+![Overview of the sky tokens and conversion module](./diagrams/sky-tokens.png)
+
+## Sky Rewards
+
+A `StakingReward` contract allows users to stake `USDS` and receive `SKY` as a reward according to an issuance chosen by the governance. Users can stake their `USDS` directly in the `StakingRewards` contract and claim their rewards with the same contract. `DssVestMintable` contracts allows the governance to create vested plans that will gradually mint tokens to distribute to some beneficiaries.
```

Comment:
where does sUSDS tie in here? its currently unclear where users stake USDS to receive sUSDS and where sUSDS yield is coming from